CHANGES

```diff
@@ -3,6 +3,39 @@ repositories (changes that are automatically handled by the format upgrade tools
 are not marked). Those prefixed with "(+)" are new command/option (since
 2.1.0~alpha2).
 
+2.1.3:
+* [BUG] Fix `opam init` and `opam init --reinit` when the `jobs` variable has
+  been set in the opamrc or the current config. [#5056 @rjbou]
+* When inferring a 2.1+ switch invariant from 2.0 base packages, don't filter
+  out pinned packages as that causes very wide invariants for pinned compiler
+  packages [#5176 @dra27 - fix #4501]
+* Some optimisations to 'opam list --installable' queries combined with other
+  filters [#4882 @AltGr]
+* Improve performance of some opam list combination (e.g. --available
+  --installable) [#4999 @kit-ty-kate]
+* Improve performance of opam list --conflicts-with when combined with other
+  filters [#4999 @kit-ty-kate]
+* Improve performance of opam show by 300% when the package to show is given
+  explicitly or unique [#4998 @kit-ty-kate - fix #4997 and partially #4172]
+* Don't error when displaying if switch is not set [#5027 @rjbou - fix #5025]
+* Try to set a variable with option `--switch <sw>` fails instead of writing a
+  wrong `switch-config` file [#5027 @rjbou] * When a field is defined in switch
+  and global scope, try to determine the scope also by checking switch
+  selection [#5027 @rjbou]
+* Stop zypper from upgrading packages on updates on OpenSUSE [#4978
+  @kit-ty-kate]
+* Print error message if command doesn't exist [#4971 @kit-ty-kat - fix #4112]
+* Actually allow multiple state caches to co-exist [#4934 @dra27 - fix #4554
+  properly this time]
+* Update cold compiler to 4.13 [#5017 @dra27]
+* Bump opam-file-format to 2.1.4 [#5117 @kit-ty-kate - fix #5116]
+* Fix (at least some of the) empty conflict explanations [#4982 @kit-ty-kate]
+* Add some tests [#4841 #4974 #4861 #4915 #4979 #5004 #5006 #5015 #5024 #5025
+  #5031 #5131 #5176 @AltGr @dra27 @kit-ty-kate]
+* Update test engine [#4913 #4966 #4979 #5004 #5009 #5024 #5097 @AltGr
+  @kit-ty-kate @rjbou]
+* Update documentation [#5097 @rjbou]
```

Comment:
```suggestion
* [BUG] Fix `opam init` and `opam init --reinit` when the `jobs` variable has
  been set in the opamrc or the current config. [#5056 @rjbou]
* When inferring a 2.1+ switch invariant from 2.0 base packages, don't filter
  out pinned packages as that causes very wide invariants for pinned compiler
  packages [#5176 @dra27 - fix #4501]
* [BUG] Fix an internal error on repository upgrade from OPAM 1.2 [#4965 @AltGr]
* Some optimisations to `opam list --installable` queries combined with other
  filters [#4882 @AltGr - fix #4311]
* Improve performance of some opam list combinations (e.g. `--available`,
  `--installable`) [#4999 @kit-ty-kate]
* Improve performance of `opam list --conflicts-with` when combined with other
  filters [#4999 @kit-ty-kate]
* Improve performance of `opam show` by as much as 300% when the package to show is given
  explicitly or is unique [#4998 @kit-ty-kate - fix #4997 and partially #4172]
* [BUG] `opam var` no longer fails if no switch is set [#5027 @rjbou - fix #5025]
* [BUG] Setting a variable with option `--switch <sw>` fails instead of writing an
  invalid `switch-config` file [#5027 @rjbou]
* When a field is defined in switch and global scope, try to determine the
  scope also by checking switch selection [#5027 @rjbou]
* Stop Zypper from upgrading packages on updates on OpenSUSE [#4978
  @kit-ty-kate]
* Clearer error message if a command doesn't exist [#4971 @kit-ty-kat - fix #4112]
* Actually allow multiple state caches to co-exist [#4934 @dra27 - actually fixes #4554]
* Update cold compiler to 4.13 to avoid issues with glibc 2.34 on Unix [#5017 @dra27]
* Bump opam-file-format to 2.1.4 [#5117 @kit-ty-kate - fix #5116]
* Fix some empty conflict explanations [#4982 @kit-ty-kate - partially fix #4373]
* [BUG] Fix `make doc` with Dune 2.7 (unblocks `opam2web` build) [#5030 @kit-ty-kate]
* Port some tests from master [#4841 #4974 #4861 #4915 #4979 #5004 #5006 #5015 #5024 #5025
  #5031 #5131 #5176 @AltGr @dra27 @kit-ty-kate]
* Update test engine to allow for additional tests [#4913 #4966 #4979 #5004 #5009 #5024 #5097 @AltGr
  @kit-ty-kate @rjbou]
* Update for git protocol deprecation on GitHub [#5097 @rjbou]
```